config.go

```diff
@@ -113,6 +114,23 @@ func init() {
 	if err != nil {
 		log.Fatalf("failed to load kata configuration: %v\n", err)
 	}
+
+	EnvKataHypervisor := os.Getenv("KATA_HYPERVISOR")
```

Comment:
I think we should pass this information *explicitly* as an option, see [Makefile](1)

[1]: https://github.com/kata-containers/tests/blob/master/Makefile#L101

Reply:
Can you please elaborate more about what's the right way to pass the information of `KATA_HYPERVISOR` to the ginkgo tests? Thanks a lot.

@chavafg @GabyCT @jodh-intel

Reply:
Custom Ginkgo CLI options (like `--runtime=` and `--timeout=`) are passed to the `ginkgo` binary after the "`--`" separator. See for example:

- https://github.com/kata-containers/tests/blob/master/Makefile#L101

And those options are defined here:

- https://github.com/kata-containers/tests/blob/master/command.go#L32..L33

Hence, you could add a new `--hypervisor=` option so that users / the CI could run:

```
$ make HYPERVISOR=foo
```

Which would result in the `Makefile` calling `ginkgo ... -- --hypervisor=foo ...`.